University-advisor/Services/EmailSenderService.cs

```diff
@@ -31,11 +32,11 @@ public static void SendEmail(string recipient, string subject, string message)
                 msg.Subject = subject;
                 msg.Body = message;
                 SendThroughSmtp(msg);
-                Logger.Log("Email was successfully sent.");
+                Logger.Log(Messages.sendEmailSuccess);
```

Comment:
Could you export client set up strings to Constant directory as well? I am talking about 18, 19 lines.

Reply:
Yeah, sure. good idea